src/main/app/claims/responseModelConverter.ts

```diff
@@ -43,7 +43,7 @@ export class ResponseModelConverter {
   // TODO A workaround for Claim Store staff notifications logic to work.
   // Should be removed once partial admission feature is fully done and frontend and backend models are aligned properly.
   private static inferDefenceType (draft: ResponseDraft): DefenceType {
-    if (draft.response.type === FormResponseType.OWE_NONE) {
+    if (draft.response.type === FormResponseType.DEFENCE) {
```

Comment:
In the backend we will have `FULL_DEFENCE` instead of `DEFENCE` for now. It would be good to inline these at some point.